packages/client/src/utils.test.ts

```
@@ -27,3 +27,218 @@
+    let cleanup = false
+    const iterator = (async function* () {
+      try {
+        await new Promise(resolve => setTimeout(resolve, 25))
+        yield 1
+        yield 2
+        return 3
+      }
+      finally {
+        cleanup = true
+        // eslint-disable-next-line no-unsafe-finally
+        throw error
```

Biome (2.1.2)

[error] Unsafe usage of 'throw'.

'throw' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)